### PR TITLE
June Newsletter 

### PR DESCRIPTION
**Pull request by MxRaCohen**
Complete with ad stats for June, server updates, and what we're working on next

### CODE REVIEW

File: content/posts/newsletter-june-2021.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+* We had **14,574,934** paid ad views across the world.
+
+Reminder, we are actively working to onboard
+new, high quality publishers, which will expand our ad inventory.
+
+
+## Tricky bits
+
+We are still verging on max capacity with regards to ad inventory.
+Marketing and growth is a challenge for any small business.
```
Comment (ericholscher) [inline content/posts/newsletter-june-2021.md:43]: Didn't we use this exact line last month?

File: content/posts/newsletter-june-2021.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+To address this, we rolled out a number of new house ads across the network.
+Unfortunately, determining the success of these new ads is difficult
+without tracking users or conversions.
```
Comment (ericholscher) [inline content/posts/newsletter-june-2021.md:46]:
```suggestion
without using common third-party tools that track users.
```

File: content/posts/newsletter-june-2021.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+We are looking into privacy preserving analytics options for our website.
+
+
+## Upcoming features
+
+The major features in our upcoming roadmap:
+
+* Adding a staff interface for customizing ad flights is currently our top priority.
```
Comment (ericholscher) [inline content/posts/newsletter-june-2021.md:54]:
```suggestion
* Adding a staff interface for customizing ad flights.
```

Feels weird to say this is our top priority. I don't think it is.

File: content/posts/newsletter-june-2021.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+* We are still in the process of designing a referral system for our current publishers.
```
Comment (ericholscher) [inline content/posts/newsletter-june-2021.md:55]:
```suggestion
* We are still not sure the best way to implement referrals.
  We might try something simple to see if it works, and build on it if we find that it works. 
```

**Review by ericholscher — APPROVED** [review]
Good overview..

File: content/posts/newsletter-june-2021.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+
+## New features
+
+These are the major features we added in the last month:
+
+* In our ad server, we updated the weighting algorithm to slightly bias showing CPC ads on higher CTR publishers.
+* Our ad server now tracks historical changes to ad campaigns.
+* Also in our ad server, we added actions to allow our staff to handle
+  adding new publishers and execute publisher payouts.
```
[the remark this answers is not in the document]
Comment (davidfischer) [reply content/posts/newsletter-june-2021.md:17]: We always handled payouts so I wonder if we should just talk about "streamlining publisher payouts".

File: content/posts/newsletter-june-2021.md (hunk 1 above)
Comment (davidfischer) [reply content/posts/newsletter-june-2021.md:43]: Maybe we should briefly mention that we raised prices in Q3 especially in North America where we are at capacity. I think the honesty will be appreciated and publishers will like knowing that we're trying to make them more money.

File: content/posts/newsletter-june-2021.md (hunk 3 above)
Comment (davidfischer) [reply content/posts/newsletter-june-2021.md:54]: Ya, I think we should frame this as "streamlining how we setup flights for advertisers". This also encapsulates a few other changes like automated re-ups and a campaign builder.

File: content/posts/newsletter-june-2021.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (MxRaCohen) [reply content/posts/newsletter-june-2021.md:18]: We didn't have staff handle publisher payouts however.

File: content/posts/newsletter-june-2021.md (hunk 1 above)
Comment (MxRaCohen) [reply content/posts/newsletter-june-2021.md]: I mentioned the price increase and removed the duplicated line

**Review by davidfischer — APPROVED** [review]

**Pull request merged**